docs/management_api.yml

```diff
@@ -298,6 +408,26 @@ components:
         error: "<error description>"
         request_id: "eed14d55-d996-42cd-8248-e806663810a8"
 
+    FileUpload:
+      type: object
+      properties:
+        path:
+          type: string
+          description: The path of the file on the device
+        uid:
+          type: integer
+          description: The numerical UID of the file on the device
```

Comment:
I would explicitly note that this UID is the local user id on the device and has no meaning outside it. Also for these reasons I have doubts if it s worthwhile to return the ids, for the UI it would be of some value to get the username and the name of the group, if we want to present it. otherwise we need another protocol message GETUID/GETGID to translate.

Reply:
I don't think we should increase the complexity and implement getUID/getGID or similar. The issue is that you won't be able to set non-numerical owners to the local file anyway, unless you have them in your local system. @alfrunes's idea was to stick to the numerical IDs because at least they are portable, and this is what you do when you us tar with `--numeric-owner`. What shall we do here?

Reply:
when you pass a username and a group name to the mender-connect in the message which creates a file, then locally you resolve to uid and gid to set the owner/group on the local file. passing and storing the ids of a local system outside of it seems a bit strange.
and it is no accident that tar has `--numeric-owner` it is exactly this use case.

Reply:
but if you both say it is ok, then I will not argue too much.

Reply:
I tried to keep the implementation at the filesystem level, without involving OS specific user management. I think you can use the same argument with OS user/group names; they don't have any meaning outside the device's OS, and may have different numerical UIDs/GIDs across different devices.
I'm open to change it if it's a bad idea, but these were my thoughts while developing this.

Reply:
we won't change this at this stage